modules/creating-custom-live-rhcos-iso.adoc

```diff
@@ -22,67 +22,36 @@ You can create a live {op-system} ISO with SSHd enabled and with predefined cred
 
 . Create the `embedded.yaml` file that the `butane` utility uses to create the Ignition file:
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
-variant: rhcos
-version: 0.1.0
+variant: openshift
+version: {product-version}.0
+metadata:
+  name: sshd
+  labels:
+    machineconfiguration.openshift.io/role: worker
 passwd:
   users:
-    - name: root
+    - name: core <1>
       ssh_authorized_keys:
         - '<ssh_key>'
-      password_hash: '$2b$05$aAd9TkvWnvUmeDQEA9vkiOoCqCZgd6cWn0xXwe.ii/TqeCxUC0Z0G' <1>
-storage:
-  files:
-    - path: /etc/ssh/sshd_config
-      contents:
-        local: ./live-sshd_config
-      mode: 0600
-      overwrite: true
 ----
-<1> `yescrypt` passwords are not supported in {op-system}. Use `bcrypt` instead when you create the `password_hash`.
+<1> The `core` user has sudo privileges.
```

Comment:
Has or must have?

Reply:
Has 🙂 coreos `core` user comes with special admin privs by default, but still needs to be configured for actual use with ssh keys etc. just like you'd configure other users, so it's worth calling out here to note differences